odl/contrib/fom/supervised/supervised_fom.py

```diff
@@ -0,0 +1,575 @@
+
+    .. math::
+        \\frac{\| f - g \|^2_2}{\| f \|^2_2 + \| g \|^2_2}.
+
+    The normalized FOM takes its values in [0, 1], with higher correspondance
+    at lower FOM value.
+
+    Parameters
+    ----------
+    data : `FnBaseVector`
+        Input data or reconstruction.
+    ground_truth : `FnBaseVector`
+        Reference to compare ``data`` to.
+    mask : `FnBaseVector`
+        Binary mask to define ROI in which FOM evaluation is performed.
+    normalized  : bool
+        Boolean flag to switch between unormalized and normalized FOM.
+
+    Returns
+    -------
+    fom : float
+        Scalar (float) indicating mean squared error between ``data`` and
+        ``ground_truth``. In normalized form the FOM takes values in
+        [0, 1], with higher correspondance at lower FOM value.
+    """
+
+    l2_normSquared = odl.solvers.L2NormSquared(data.space)
+
+    if mask is not None:
+        data = data * mask
+        ground_truth = ground_truth * mask
+
+    diff = data - ground_truth
+    fom = l2_normSquared(diff)
+
+    if normalized:
+            fom /= (l2_normSquared(data) + l2_normSquared(ground_truth))
+
+    return fom
+
+
+def mean_absolute_error(data, ground_truth, mask=None, normalized=False):
+    """FOM returning the L1-distance between ``data`` and ``ground_truth``.
+
+    Evaluates `mean absolute error
+    <https://en.wikipedia.org/wiki/Mean_absolute_error>`_ between
+    input (``data``) and reference (``ground_truth``), by measuring
+    consistency using the L1-norm. Allows for normalization (``normalized``)
+    and a masking of the two spaces (``mask``).
+
+    Notes
+    ----------
+    The FOM evaluates
+
+    .. math::
+        \| f - g \|_1,
+
+    or, in normalized form
+
+    .. math::
+        \\frac{\| f - g \|_1}{\| f \|_1 + \| g \|_1}.
+
+    The normalized FOM takes its values in [0, 1], with higher correspondance
+    at lower FOM value.
+
+    Parameters
+    ----------
+    data : `FnBaseVector`
+        Input data or reconstruction.
+    ground_truth : `FnBaseVector`
+        Reference to compare ``data`` to.
+    mask : `FnBaseVector`
+        Binary mask to define ROI in which FOM evaluation is performed.
+    normalized  : bool
+        Boolean flag to switch between unormalized and normalized FOM.
+
+    Returns
+    -------
+    fom : float
+        Scalar (float) indicating mean absolute error between ``data`` and
+        ``ground_truth``. In normalized form the FOM takes values in
+        [0, 1], with higher correspondance at lower FOM value.
+    """
+
+    l1_norm = odl.solvers.L1Norm(data.space)
+    if mask:
+        data = data * mask
+        ground_truth = ground_truth * mask
+    diff = data - ground_truth
+    fom = l1_norm(diff)
+
+    if normalized:
+        fom /= (l1_norm(data) + l1_norm(ground_truth))
+
+    return fom
+
+
+def mean_value_difference(data, ground_truth, mask=None, normalized=False):
+    """FOM returning the  difference in mean value between ``data``
```

Comment:
Since it's a function returning something, we tend to use `Return ...`.
Regarding FOM, agree, it's part of the `fom` package so it should be obvious that it's a FOM.